### ISSUE TITLE
Provide model.dropCollection

### ISSUE THREAD

**Issue by AV25242** [couchbaselabs/node-ottoman#392]
Provide ability to drop a collection at a model level such that the name of the scope and collection are auto inferred if not provided explicitly

**Comment by httpJunkie** [comment]
Blocked by:
https://github.com/bwgjoseph/mongoose-vs-ottoman/issues/4

**Comment by AV25242** [comment]
Even though the issue on the customer side is still open, the latest comments indicates that these issues can be closed. We may open a new issue as needed
